screenpy_playwright/actions/refresh_the_page.py

```diff
@@ -0,0 +1,50 @@
+"""Refresh the page!"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Literal, TypedDict
+
+from screenpy import UnableToAct, beat
+
+from screenpy_playwright.abilities import BrowseTheWebSynchronously
+
+if TYPE_CHECKING:
+    from screenpy import Actor
+    from typing_extensions import NotRequired, Unpack
+
+    class ReloadTypes(TypedDict):
+        """Types that can be passed to Playwright's Page.reload."""
+
+        timeout: NotRequired[float | None]
+        wait_until: NotRequired[
+            Literal["commit", "domcontentloaded", "load", "networkidle"] | None
+        ]
+
+
+class RefreshThePage:
+    """Refresh the current page in the browser.
+
+    Examples:
+        the_actor.attempts_to(RefreshThePage())
+    """
+
+    def __init__(self, **kwargs: Unpack[ReloadTypes]) -> None:
+        """Initialize the action."""
+        self.kwargs = kwargs
```

Comment:
Is there an example of kwargs that could go in the docstring?

Reply:
I think the approach sounds good.  Having a common use case example would be good; especially if each action might have different `ReloadTypes` to utilize.  

I also suggest specifically typing the instance attribute `kwargs`.  While mypy infers the type from the argument in `__init__`, other tools will work better if it's specifically typed.

example:
```
class RefreshThePage:
    kwargs: ReloadTypes
```

I also think it would be good to include a test for these actions with a custom kwargs for each.

Reply:
The `kwargs` we're talking about are just passed down to Playwright, so there's nothing in our code that will be affected by those `kwargs`. I guess a test to make sure the `kwargs` reach their destination would be useful.